docs/error-reporting/project-setup/submission-actions.md

```diff
@@ -36,8 +36,7 @@ This action can be used for automated assignment or to set specific attribute va
 
 <img src={useBaseUrl('img/error-reporting/project-settings/submission-action-set-value.png')} alt="" />
 
-
 ### Union Values
 This action is like Set Values, but for ‘label’ types, like Tags or Classifiers, where instead of ‘setting’ a value, the system is ‘adding’ a value to the field, so it can contain multiple values.
```

Comment:
Not sure single quotes are needed here? 
```suggestion
This action is like Set Values, but for label types, like Tags or Classifiers, where instead of setting a value, the system is adding a value to the field, so it can contain multiple values.
```